### ISSUE TITLE
Remember Last Script per Server

### ISSUE THREAD

**Issue by biomagnet** [Fate-Grand-Automata/FGA#928]
### Preparation

- [X] No one has asked for this feature before in other issues (even the closed ones)

### Describe the feature

As of right now, it seems that internally the app still considers the list of scripts to be all in one and then chooses to only display the ones that apply to the current server. What this means is that if I farm on JP until AP runs out, then go to NA, I have to re-select the NA script (because it is remembering its position on the JP script). If I finish on NA and then come back later to farm on JP, again the app doesn't know what script to sit on because it's remembering the last NA one.

So, basically, some way to mark the last remembered script as different between servers.
